src/native/external/brotli-version.txt

```diff
@@ -1,2 +1,6 @@
-1.0.9
-https://github.com/google/brotli/releases/tag/v1.0.9
+1.1.0
+https://github.com/google/brotli/releases/tag/v1.1.0
+
+Copy the c/dec, c/enc, c/common, and c/include folders into the root and remove all other files.
```

Comment:
We should probably create a script to do this, if we are going to selectively copy stuff. Previously, the idea was to keep these external dependencies as-is, so it's easy to compare/review during the updates.

Reply:
I'm planning (and have a branch locally that I need to update) that will bring us in alignment with how we handle other dependencies.